core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -148,8 +148,19 @@ public void close() {
             return;
         }
 
+        boolean connectionIsLive = false;
+
+        try {
+            connectionIsLive = !connection.isClosed();
+        } catch (SQLException e) {
+            // if the connection state can not be determined, assume that the
+            // connection is closed
+            suppressed.add(e);
```

Comment:
If we add a thrown exception to `suppressed` here, won't it get thrown later on after we call `closer.close`? Then it would be thrown anyway when it seemed like the intent is that `close()` shouldn't throw in that case.

Reply:
that is a good point. I guess we should just drop the exception then. It will be a "connection closed" or "IO exception" anyway.